packages/react-dom-bindings/src/client/ReactDOMUpdatePriority.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import type {EventPriority} from 'react-reconciler/src/ReactEventPriorities';
+
+import {NoEventPriority} from 'react-reconciler/src/ReactEventPriorities';
+
+let currentUpdatePriority: EventPriority = NoEventPriority;
+
+export function setCurrentUpdatePriority(newPriority: EventPriority): void {
+  currentUpdatePriority = newPriority;
+}
+
+export function getCurrentUpdatePriority(): EventPriority {
+  return currentUpdatePriority;
+}
+
+export function runWithPriority<T>(priority: EventPriority, fn: () => T): T {
```

Comment:
It seems like maybe this was kind of broken to begin with since it doesn't opt out of transitions.

Reply:
well it's sorta legacy anyway. Aren't we going to remove from OSS entrypoints?